### DISCUSSION TITLE
The prop 'isSelected' of Tab

### DISCUSSION THREAD

**Discussion by Wacheer** [chakra-ui/chakra-ui#5425] [Q&A] [answered]
When I use 'isSlected' on Tab , there is a warning appeared. 

Warning:
 "Warning: React does not recognize the `isSelected` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `isselected` instead. If you accidentally passed it from a parent component, remove it from the DOM element."

my code:
``` javascript
<Tbs>
      <TabList flexDirection={props.direction}>
        {props.options.map((option) => {
          return (
            <Tab
              _focus={{ boxShadow: "none" }}
              key={option.id}
              isSelected={option.isActive}
            >
              {option.label}
            </Tab>
          );
        })}
      </TabList>
    </Tbs>
```

**Comment by primos63** [comment, marked as answer]
There is no prop `isSelected` on `Tab`. The docs erroneously show the prop, but it is not valid. What you are looking for is a controlled  component.  See the [Controlled Example](https://chakra-ui.com/docs/disclosure/tabs#controlled-tabs) in the docs.

```jsx
function ControlledTabs(props) {
  const activeIndex = props.options.findIndex(option => option.active)
  const [tabIndex, setTabIndex] = React.useState(activeIndex)   // assumes at least one of the options is active

  const handleTabsChange = (index) => {
    setTabIndex(index)
  }

  return (
    <Tabs index={tabIndex} onChange={handleTabsChange}>
        <TabList flexDirection={props.direction}>
          {props.options.map((option) => {
            return (
              <Tab
                _focus={{ boxShadow: "none" }}
                key={option.id}
              >
                {option.label}
              </Tab>
            );
          })}
        </TabList>
    </Tabs>
  )
}
```

**Update**
This looks to be both a docs and code issue. `isSelected` and `panelId` are listed as valid props for `Tab`; however, they are not use in the code.
```typescript
export interface UseTabOptions {
  id?: string
  isSelected?: boolean
  panelId?: string
  /**
   * If `true`, the `Tab` won't be toggleable
   */
  isDisabled?: boolean
}
```
should be
```typescript
export interface UseTabOptions {
  id?: string
  /**
   * If `true`, the `Tab` won't be toggleable
   */
  isDisabled?: boolean
}
```

**Reply by Wacheer** [reply]
oh, thanks! it's helpful

**Reply by guizmaii** [reply]
That'd be simpler if Chakra was implementing the `isSelected` prop, no?

**Comment by roqueheli** [comment]
Genius, thanks a lot. I had been searching this solution for several days!!
